rpc/packetizer.go

```diff
@@ -44,21 +58,21 @@ func (p *packetHandler) NextFrame() (rpcMessage, error) {
 
 	// Interpret the byte as the length field of a fixarray of up
 	// to 15 elements: see
-	// https://github.com/msgpack/msgpack/blob/master/spec.md#formats-array
-	// . Do this so we can decode directly into the expected
-	// fields without copying.
+	// https://github.com/msgpack/msgpack/blob/master/spec.md#formats-array .
```

Comment:
`s/array ./array.`

Reply:
I try not to end a sentence with a URL and punctuation, since the punctuation can be interpreted as part of the URL. Changed the wording slightly instead.